config/discourse_defaults.conf

```diff
@@ -229,6 +229,9 @@ s3_endpoint =
 s3_http_continue_timeout =
 s3_install_cors_rule =
 
+# Optionally, specify a separate CDN to be used for static JS assets stored on S3
+s3_asset_cdn_url =
```

Comment:
If this is only used for JS assets, I was wondering if the setting name can reflect that since asset may refer to CSS files too.

Reply:
It relates to all 'static assets' on S3. Stylesheets are not currently on S3, but they might be in future, so I'd prefer to keep it generic.